src/components/AppShelf/AppShelfItem.tsx

```diff
@@ -134,7 +134,7 @@ export function AppShelfItem(props: any) {
                 </Anchor>
               </motion.i>
             </AspectRatio>
-            <PingComponent url={service.url} status={service.status} />
+            {service.ping !== false && <PingComponent url={service.url} status={service.status} />}
           </Card.Section>
```

Comment:
Is there a reason, why you use ``!== false`` instead of ``== true`` or ``service.ping &&``?

Reply:
In an effort to stay consistent with how the current config is handled, I've made it so that the `service.ping` property only appears in the config if it is set to `false`. If the user wants their service to be pinged (and sets "Ping Service" to true), there will be no `ping` property set in their config. This might be able to be re-written to simply check if `service.ping` exists (because if it does, it means it was set to false). 

Please let me know what you think.

Reply:
Good idea , thanks for the changes I didn't think about that

Reply:
Ok, great. You can resolve this conversation then :)

Reply:
Thank you both for the feedback :)